maquetado/scss/tech.scss

```diff
@@ -8,28 +8,22 @@
 .tech {
   @extend %equal-align;
 
-  background-color: $green;
   border-radius: 5px;
```

Comment:
this is not necessary and its making the green line looks with radius in the edges

Reply:
in fact, you dont need `position:relative` and `margin` :) check the new design and just use the padding and the grid gap properties